p5js/ImageClassification/ImageClassification_VideoScavengerHunt/index.html

```diff
@@ -0,0 +1,32 @@
+<!--
+ Copyright (c) 2018 ml5
+ 
+ This software is released under the MIT License.
+ https://opensource.org/licenses/MIT
+-->
+
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Webcam Image Classification with Speech Output using MobileNet, p5.js, p5.speech</title>
+
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/addons/p5.dom.min.js"></script>
+  <script src="https://unpkg.com/ml5@0.1.2/dist/ml5.min.js" type="text/javascript"></script>
+  <script src="lib/p5.speech.js"></script>
```

Comment:
Is there a CDN for p5 speech?

Reply:
I couldn't find any public link for p5 speech..